include/class.forms.php

```diff
@@ -1760,20 +1771,14 @@ function toString($value) {
         $config = $this->getConfiguration();
         // If GMT is set, convert to local time zone. Otherwise, leave
         // unchanged (default TZ is UTC)
+        if (!$value)
+            return '';
         if ($config['time'])
             return Format::datetime($value, false, !$config['gmt'] ? 'UTC' : false);
         else
             return Format::date($value, false, false, !$config['gmt'] ? 'UTC' : false);
     }
 
-    function export($value) {
-        $config = $this->getConfiguration();
-        if (!$value)
-            return '';
-        else
-            return Format::date($value, false, 'y-MM-dd HH:mm:ss', !$config['gmt'] ? 'UTC' : false);
-    }
-
```

Comment:
I'm thinking the CSV export should use a more computer-readabale date format (like the one in the red block). Otherwise, the locale-based format from `Format::datetime()` would be used, and spreadsheet programs might not parse the date well. Is there a standard on this?

Unpatched databases won't benefit from the new code. However, the new code will break stuff too. One line could be changed in `class.misc.php`, around line 96 in `dbtime`
```php
            $time = is_int($var) ? $var : strtotime($var) ?: $var;
```
which would allow defaulting back to `$var` (assuming its a string Unix timestamp) if `$var` is neither a PHP integer nor a valid date format. This would allow unpatched databases to keep the existing data which recording new and updated data in the new format. New and update data would also be searchable. Perhaps we could cook something up for the command-line for manual patching